compiler/src/AST/Source.hs

```diff
@@ -176,14 +170,14 @@ data Port = Port (A.Located Name) Type
 
 data Effects
   = NoEffects
-  | Ports [(SourceOrder, Port)]
-  | Manager A.Region Manager
+  | Ports [(SourceOrder, Port)] SC.PortsComments
+  | Manager A.Region Manager SC.ManagerComments
```

Comment:
💭 another option would be to have these import as `Comments.Manager` (having the type name `Manager` overloaded but expect that it's always used via qualified import)

Reply:
Importing as `Comments.Manager` seems like a better option to me, but I'm fine with how it looks now as well.

Reply:
👍 I'll save considering this for a future PR, then